chore: LEAP-96: mark all datasets endpoints internal

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/17627583488)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/565
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17478044822

jombooth commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17478285294)
> Successfully merged: Already up to date.

jombooth commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17623419096)
> Successfully merged:  225 files changed, 7635 insertions(+), 2771 deletions(-)

jombooth committed:
Merge branch 'develop' into 'fb-ROOT-96/mothball-datasets-endpoints'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17623419096

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17624523503

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17625111657

jombooth commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17625210357)
> Successfully merged: Already up to date.

jombooth committed:
Merge branch 'develop' into fb-ROOT-96/mothball-datasets-endpoints

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17625679264

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17626069864

jombooth committed:
Merge branch 'develop' into fb-ROOT-96/mothball-datasets-endpoints

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17627503597

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17627583488

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/17628623357
